tdrs-backend/tdpservice/users/migrations/0012_add_initial_super_user.py

```diff
@@ -0,0 +1,55 @@
+import os
+import logging
+from django.contrib.auth.hashers import make_password
+from django.db import migrations
+from django.utils import timezone
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class NoSuperUser(Exception):
+    pass
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('users', '0011_auto_20210108_1741'),
+    ]
+
+    def generate_superuser(apps, schema_editor):
+        # raise an exception if the environment variable isn't set
+        if os.environ.get('DJANGO_SU_NAME') is None:
+            raise NoSuperUser("DJANGO_SU_NAME does not have a value set.")
+
+        # set the environment variable to the username of the
+        # initial superuser
+        su_username = os.environ.get('DJANGO_SU_NAME')
+
+        # Get current time for date_joined
+        now = timezone.now()
+
+        # Sets a password the user won't be able to log in with
+        # Needed because we defer to Login.gov for authentication
+        # and users mustn't be allowed to login directly.
+        unusable_password = make_password(None)
```

Comment:
`set_unusable_password()` is called when creating a new user from login.gov (https://github.com/HHS/TANF-app/blob/ebea95e25f7e0250c46f1d7f6777efdd84f8152b/tdrs-backend/tdpservice/users/api/login.py#L67). is it needed here too?

Reply:
We had done it that way before, but migrations in django do not work with custom methods on models. We have to use historical models, check the comment below. This migration would fail when using that method.

Reply:
This is because the historical models don't serialize model methods so anything outside of the manager is unavailable to the migration. 

set_unusable_password just calls: `self.password = make_password(None)`

We can't use set_unusable_password with the historical model because its a method on User but make_password is an external import not tied to the model directly so we can use it in place